tests/integration/tests/conftest.py

```diff
@@ -48,7 +49,8 @@ def h() -> harness.Harness:
 def pytest_configure(config):
     config.addinivalue_line(
         "markers",
-        "node_count: Mark a test to specify how many instance nodes need to be created",
+        "node_count: Mark a test to specify how many instance nodes need to be created\n"
+        + "etcd_count: Mark a test to specify how many etcd instance nodes need to be created (None by default)",
```

Comment:
python doesn't mind concatenation of strings split over multiple lines, but the linter might hate it
```
>>> print(
... "i am \n"
... "a multiline string"
... )
i am 
a multiline string
```

```suggestion
        "node_count: Mark a test to specify how many instance nodes need to be created\n"
        "etcd_count: Mark a test to specify how many etcd instance nodes need to be created (None by default)",
```